LingoClash/LingoClash/DataProviders/DataProviderTypes.swift

```diff
@@ -13,11 +13,16 @@ protocol Record: Codable {
     var id: Identifier { get }
 }
 
+extension Record {
+    // Local creation of Records to be pushed to DataProvider where id is useless
```

Comment:
As described this placeHolderId is used for the creation of Records locally to then be created/ used for update in DataManager. Id cannot be blank